### DISCUSSION TITLE
Release 5.3.5

### DISCUSSION THREAD

**Discussion by j0k3r** [serverless-heaven/serverless-webpack#1552] [Team Posts]
If you all agree, I'll release the 5.3.5 with that PR:
- https://github.com/serverless-heaven/serverless-webpack/issues/633

And I'll also create a new one to upgrade all deps so we'll be up to date.
Anything else we should put in that patch release?

**Comment by miguel-a-calles-mba** [comment]
Sounds good to me.

What about adding this one? https://github.com/serverless-heaven/serverless-webpack/pull/629

**Comment by j0k3r** [comment]
Added & merged!

**Comment by j0k3r** [comment]
Can someone take a look at my current PRs https://github.com/serverless-heaven/serverless-webpack/pull/636 & https://github.com/serverless-heaven/serverless-webpack/pull/637?
So I'll be able to release 5.3.5, thanks!

**Comment by miguel-a-calles-mba** [comment]
Sure thing. They are approved.

**Comment by j0k3r** [comment]
PR for the release is ready https://github.com/serverless-heaven/serverless-webpack/pull/641

**Comment by miguel-a-calles-mba** [comment]
Nice! The npm package will automatically publish after publishing the release version.
